src/Components/Card.jsx

```diff
@@ -1,18 +0,0 @@
-import Image from 'next/image'
-
-const Card = ({className,img,heading,para }) => {
```

Comment:
please be descriptive with the variables 

cardHeader and cardBody are better names for this